### PR TITLE
Fixup tutorial on creating theme from existing HTML templates

### PR DESCRIPTION
**Pull request by ashmaroli**
Found a few errors in the latest tutorial some of which, we missed while reviewing, the others became evident on reading the generated output.

/cc @jekyll/documentation @tomjohnson1492

### CODE REVIEW

File: docs/_tutorials/create-jekyll-theme.md [hunk 1]
```diff
@@ -147,7 +147,7 @@ Some page content here...
 Create another page for testing called `about.md` with similar front matter tags.
 
 {: .note .info}
-**Note:** If you don't specify a layout in your pages, Jekyll will automatically use the template labeled `default`. We specify it here only to make it explicit what's happening.
+If you don't specify a layout in your pages, Jekyll will simply render that page as an unstyled basic HTML page.
```
Comment (ashmaroli) [inline docs/_tutorials/create-jekyll-theme.md]: Jekyll doesn't automatically assign a layout. GitHub Pages does it via a [plugin](https://github.com/benbalter/jekyll-default-layout).

**Comment by ashmaroli** [comment]
@tomjohnson1492 i'd also like to make some edits to the graphic illustration. Will you be able to send the source file?

**Comment by tomjohnson1492** [comment]
Looks great. Thanks for making these edits to improve the content. This is one of the enjoyable parts about collaborating on documentation. Re the source file, I attached it here:
[jekylllayoutconcept.ai.zip](https://github.com/jekyll/jekyll/files/895203/jekylllayoutconcept.ai.zip)

You can add it to the project if you want, but as a binary file it won't play well in a repo. Also, note that this is an .ai file (Adobe Illustrator) so you'll need Illustrator or something else that can read .ai files. I can convert this to SVG if you prefer that format.

**Comment by tomjohnson1492** [comment]
@ashmaroli In case you want SVG instead of AI, it's attached here: [jekylllayoutconcept.svg.zip](https://github.com/jekyll/jekyll/files/895216/jekylllayoutconcept.svg.zip)

**Comment by ashmaroli** [comment]
@tomjohnson1492 Thanks for the source files. I can work with AI.. no probs..

**Comment by DirtyF** [comment]
@ashmaroli Thanks for adapating the illustration to Jekyll's styleguide ❤️ 

If you're still working on this, I would like to change the title (and url) to something like "Convert an existing HTML template for Jekyll". Creating a Jekyll theme is the way you package then your files into a gem (it might be a good idea to link both notions btw).

**Comment by ashmaroli** [comment]
@DirtyF How does the following sound?
```
Convert existing HTML Sites to Jekyll Templates
```

**Comment by tomjohnson1492** [comment]
That was actually similar to my original title. I had changed it due to a suggestion in an earlier review. I like the title you propose.

**Comment by DirtyF** [comment]
@ashmaroli 👍  fine by me, as long as we avoid the confusion with 💎 -based themes.

@tomjohnson1492 😖

File: docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md [hunk 2]
```diff
@@ -51,19 +57,19 @@ layout: default.html
...
 
-This is a Jekyll site at the most basic level. Here's what is happening:
+This is a Jekyll site at the most basic functional level. Here's what is happening:
 
 * The `_config.yml` file contains settings that Jekyll uses as it processes your site. An empty config file will use default values for building a Jekyll site. For example, to convert Markdown to HTML, Jekyll will automatically use the [kramdown Markdown filter](https://rubygems.org/gems/kramdown/), without any need to specify it.
 * Jekyll looks for files with [front matter tags]({% link _docs/frontmatter.md %}) (the two sets of dashed lines `---` like those in `index.md`) and processes the files (populating site variables, rendering any [Liquid](https://shopify.github.io/liquid/), and converting Markdown to HTML).
-* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tags in the layout specified (`default`) in the front matter tags.
+* Jekyll pushes the content from all pages and posts into the `{% raw %}{{ content }}{% endraw %}` tag in the layout specified (`default`) in the front matter tags.
```
Comment (DirtyF) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: Technically content is a global **variable**, who acts as a placeholder .

File: docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md [hunk 3]
```diff
@@ -1,25 +1,30 @@
...
+In many ways, any site that is currently a static site is *already* a Jekyll site. Jekyll just allows you to automate parts of the site (like inserting pages into templates, rendering lists for navigation, generating feeds and sitemaps, and more) as it processes the files.
 
-Understanding how to convert any HTML site into a Jekyll website will open your world to many more options for Jekyll themes. Instead of searching online for "Jekyll themes," you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML template as you need to, and build the output with Jekyll.
+Understanding how to convert any HTML site into a Jekyll template will open your world to many more options for Jekyll themes. Instead of searching online for "Jekyll themes," you can choose from the large variety of HTML templates for your site, quickly Jekyll-ize the HTML template as you need to, and build the output with Jekyll.
```
Comment (DirtyF) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: site -> site or template -> template but we don't convert a site into a template (a set of templates and includes maybe)

File: docs/_data/tutorials.yml [hunk 4]
```diff
@@ -4,7 +4,7 @@
   - navigation
   - orderofinterpretation
   - custom-404-page
-  - create-jekyll-theme
+  - convert-existing-html-sites-to-jekyll-templates
```
Comment (DirtyF) [inline docs/_data/tutorials.yml]: Can we stay generic here? `convert-existing-site-to-jekyll`

File: docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md [hunk 5]
```diff
@@ -1,25 +1,30 @@
 ---
 layout: tutorials
-permalink: /tutorials/create-jekyll-theme/
-title: Create your first Jekyll theme
+permalink: /tutorials/convert-existing-html-sites-to-jekyll-templates/
```
Comment (DirtyF) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: same as above please let's keep this generic `convert-existing-site-to-jekyll`

File: docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md (hunk 2 above)
Comment (ashmaroli) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: True. This prompts me to carve out a separate section:
```
## 2. Liquid Structure

The 3 basic pillars defining Liquid language are:
  * tags -- (say what they are and what they do in one-line)
  * filters -- (say what they are and what they do in one-line)
  * variables -- (say what they are and what they do in one-line)
```
Whatsay? Or is there an appropriate page already out there that I can simply link to?

File: docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md (hunk 3 above)
Comment (ashmaroli) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: I agree.. I was really stuck as what this should be.. ultimately changed it to match the title I chose.

So what does this line finally say?
```
Understanding how to convert any site to Jekyll will open your world ...
```
?

Comment (DirtyF) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: @ashmaroli  What do you think of `Understanding how to convert HTML pages into Jekyll templates` ?

Comment (tomjohnson1492) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: Re "Understanding how to ...", in general it's bad practice to use empty gerunds in task titles. Think of what users would type in Google to find the topic. Usually users type the task they're trying to accomplish, using the simple present tense. For example, "create jekyll site from html template" or something. I recommend that phrase: "Create a Jekyll site from an HTML template" or something similar.

Comment (ashmaroli) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: @tomjohnson1492 the above suggestion was not for a title but simply a sentence in the para under review

Comment (tomjohnson1492) [inline docs/_tutorials/convert-existing-html-sites-to-jekyll-templates.md]: oh, my fault for misreading that. Sorry about that.

**Comment by DirtyF** [comment]
@tomjohnson1492 I could not resist to make some edits https://github.com/ashmaroli/jekyll/pull/1/commits/dd182f388a92bb570534fdc2e6b00f359014c70f

**Comment by DirtyF** [comment]
@jekyllbot: merge +docs

**Pull request merged**